src/bundler.js

```diff
@@ -376,6 +396,10 @@ function processCssBundle(options, cssBundle, bundleDir, cssFiles, bundleName, c
         bundleStatsCollector.AddFileHash(bundleName, allMinCss);
 
         fs.writeFile(bundleName, allCss, afterBundle);
+
+        allCssArr.forEach(function(cssFile) {
+            bundleStatsCollector.AddDebugFile(cssBundle, cssFile.path);

```

Comment:
This isn't strictly necessary since there won't ever be a `CSS` require bundle, so we could continue adding debug files one by one here. But I'm trying to keep the logic largely consistent between the `processCssBundle` and `processJsBundle` functions. At some point I'd like to just consolidate the two since they're 90% the same.